### PR TITLE
[DO NOT MERGE][Android] Remove ability to add projects manually.

### LINKED ISSUE #3565: [Draft] [Android] Remove ability to add projects manually

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
Fixes #3565

**Description of the Change**
Remove the ManualUrlInputFragment class and all associated resources.

**Release Notes**
Projects can no longer be manually added.

### CODE REVIEW

File: android/BOINC/app/src/main/res/values-eu/strings.xml [hunk 1]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Aukeratu lagundu nahi dituzun proiektu zientifikoak:</string>
   <string name="attachproject_list_header">Aukeratu proiektu bat</string>
-  <string name="attachproject_list_manual_button">Gehitu proiektua URL bidez</string>
-  <string name="attachproject_list_manual_dialog_title">Sartu proiektuaren URLa:</string>
-  <string name="attachproject_list_manual_dialog_button">Gehitu proiektua</string>
-  <string name="attachproject_list_manual_no_url">Sartu proiektuaren URLa</string>
-  <string name="attachproject_list_acctmgr_button">Gehitu kontu-administratzailea</string>
+    <string name="attachproject_list_acctmgr_button">Gehitu kontu-administratzailea</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-eu/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-fi/strings.xml [hunk 2]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Valitse projektit joihin haluat osallistua:</string>
   <string name="attachproject_list_header">Valitse projekti:</string>
-  <string name="attachproject_list_manual_button">Lisää projekti URL-osoitteella</string>
-  <string name="attachproject_list_manual_dialog_title">Anna projektin URL:</string>
-  <string name="attachproject_list_manual_dialog_button">Lisää projekti</string>
-  <string name="attachproject_list_manual_no_url">Anna projektin URL</string>
-  <string name="attachproject_list_acctmgr_button">Lisää tilihallitsija</string>
+    <string name="attachproject_list_acctmgr_button">Lisää tilihallitsija</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-fi/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Choisir un projet scientifique auquel vous voulez collaborer :</string>
   <string name="attachproject_list_header">Choisissez un projet</string>
-  <string name="attachproject_list_manual_button">Ajouter un projet via son URL</string>
-  <string name="attachproject_list_manual_dialog_title">URL du projet :</string>
-  <string name="attachproject_list_manual_dialog_button">Ajouter un projet</string>
-  <string name="attachproject_list_manual_no_url">Veuillez saisir l\'URL du projet</string>
-  <string name="attachproject_list_acctmgr_button">Choisir un gestionnaire de compte</string>
+    <string name="attachproject_list_acctmgr_button">Choisir un gestionnaire de compte</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-fr/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-lt/strings.xml [hunk 4]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Pasirinkite mokslinius projektus, prie kurių norite prisidėti:</string>
   <string name="attachproject_list_header">Pasirinkite projektą</string>
-  <string name="attachproject_list_manual_button">Pridėti projektą pagal URL</string>
-  <string name="attachproject_list_manual_dialog_title">Įveskite projekto URL:</string>
-  <string name="attachproject_list_manual_dialog_button">Pridėti projektą</string>
-  <string name="attachproject_list_manual_no_url">Prašome įvesti projekto URL</string>
-  <string name="attachproject_list_acctmgr_button">Pridėti abonemento valdiklį</string>
+    <string name="attachproject_list_acctmgr_button">Pridėti abonemento valdiklį</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-lt/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-nl/strings.xml [hunk 5]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Selecteer wetenschappelijke projecten om aan deel te nemen:</string>
   <string name="attachproject_list_header">Kies een project:</string>
-  <string name="attachproject_list_manual_button">Project toevoegen via URL</string>
-  <string name="attachproject_list_manual_dialog_title">Projectspecifieke URL invoeren:</string>
-  <string name="attachproject_list_manual_dialog_button">Project toevoegen</string>
-  <string name="attachproject_list_manual_no_url">Projectspecifieke URL invoeren</string>
-  <string name="attachproject_list_acctmgr_button">Accountmanager toevoegen</string>
+    <string name="attachproject_list_acctmgr_button">Accountmanager toevoegen</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-nl/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Selecione projetos científicos que você deseja contribuir:</string>
   <string name="attachproject_list_header">Escolha um projeto</string>
-  <string name="attachproject_list_manual_button">Adicionar projeto por URL</string>
-  <string name="attachproject_list_manual_dialog_title">Digite o URL do projeto:</string>
-  <string name="attachproject_list_manual_dialog_button">Adicionar projeto</string>
-  <string name="attachproject_list_manual_no_url">Por favor, digite o URL do projeto</string>
-  <string name="attachproject_list_acctmgr_button">Adicionar gerenciador de conta</string>
+    <string name="attachproject_list_acctmgr_button">Adicionar gerenciador de conta</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-pt-rBR/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-pt-rPT/strings.xml [hunk 7]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Selecione os projetos científicos em que deseja contribuir:</string>
   <string name="attachproject_list_header">Escolha um projeto</string>
-  <string name="attachproject_list_manual_button">Adicionar projeto por URL</string>
-  <string name="attachproject_list_manual_dialog_title">Inserir URL do projeto:</string>
-  <string name="attachproject_list_manual_dialog_button">Adicionar projeto</string>
-  <string name="attachproject_list_manual_no_url">Por favor, insira o URL do projeto</string>
-  <string name="attachproject_list_acctmgr_button">Adicionar gestor de conta</string>
+    <string name="attachproject_list_acctmgr_button">Adicionar gestor de conta</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-pt-rPT/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-zh-rTW/strings.xml [hunk 8]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">選擇您想參與的研究專案:</string>
   <string name="attachproject_list_header">選擇專案</string>
-  <string name="attachproject_list_manual_button">用網址新增專案</string>
-  <string name="attachproject_list_manual_dialog_title">輸入專案網址:</string>
-  <string name="attachproject_list_manual_dialog_button">新增專案</string>
-  <string name="attachproject_list_manual_no_url">請輸入專案網址</string>
-  <string name="attachproject_list_acctmgr_button">新增帳號管理服務</string>
+    <string name="attachproject_list_acctmgr_button">新增帳號管理服務</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-zh-rTW/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-zh-rCN/strings.xml [hunk 9]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">请选择您想贡献的科学项目：</string>
   <string name="attachproject_list_header">选择一个项目</string>
-  <string name="attachproject_list_manual_button">通过URL添加项目</string>
-  <string name="attachproject_list_manual_dialog_title">输入项目网址：</string>
-  <string name="attachproject_list_manual_dialog_button">添加项目</string>
-  <string name="attachproject_list_manual_no_url">请输入项目网址</string>
-  <string name="attachproject_list_acctmgr_button">添加项目管理器</string>
+    <string name="attachproject_list_acctmgr_button">添加项目管理器</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-zh-rCN/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-uk/strings.xml [hunk 10]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Оберіть наукові проекти, в яких ви бажаєте прийняти участь:</string>
   <string name="attachproject_list_header">Оберіть проект:</string>
-  <string name="attachproject_list_manual_button">Додати проект за адресою (URL)</string>
-  <string name="attachproject_list_manual_dialog_title">Введіть адресу (URL) проекту:</string>
-  <string name="attachproject_list_manual_dialog_button">Додати проект</string>
-  <string name="attachproject_list_manual_no_url">Будь ласка, введіть адресу (URL) проекту:</string>
-  <string name="attachproject_list_acctmgr_button">Додати менеджер проектів</string>
+    <string name="attachproject_list_acctmgr_button">Додати менеджер проектів</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-uk/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-sv/strings.xml [hunk 11]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Välj ett vetenskapligt projekt du vill bidra till:</string>
   <string name="attachproject_list_header">Välj ett projekt</string>
-  <string name="attachproject_list_manual_button">Lägg till projekt via URL</string>
-  <string name="attachproject_list_manual_dialog_title">Ange projektets URL:</string>
-  <string name="attachproject_list_manual_dialog_button">Lägg till projekt</string>
-  <string name="attachproject_list_manual_no_url">Vänligen ange projektets URL</string>
-  <string name="attachproject_list_acctmgr_button">Lägg till kontohanterare</string>
+    <string name="attachproject_list_acctmgr_button">Lägg till kontohanterare</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-sv/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-sk/strings.xml [hunk 12]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Zvoľte vedecké projekty, ktorým chcete pomôcť:</string>
   <string name="attachproject_list_header">Zvoľte projekt</string>
-  <string name="attachproject_list_manual_button">Pridať projekt pomocou URL</string>
-  <string name="attachproject_list_manual_dialog_title">Zadajte URL projektu:</string>
-  <string name="attachproject_list_manual_dialog_button">Pripojiť sa k projektu</string>
-  <string name="attachproject_list_manual_no_url">Prosím zadajte URL projektu</string>
-  <string name="attachproject_list_acctmgr_button">Pridať Správcu konta.</string>
+    <string name="attachproject_list_acctmgr_button">Pridať Správcu konta.</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-sk/strings.xml:13]: Please fix the indentation.

File: android/BOINC/app/src/main/res/values-ru/strings.xml [hunk 13]
```diff
@@ -10,11 +10,7 @@
   <!--selection list-->
   <string name="attachproject_list_desc">Выберите научные проекты, в которых Вы хотите принять участие:</string>
   <string name="attachproject_list_header">Выберите проект</string>
-  <string name="attachproject_list_manual_button">Добавить URL-адрес проекта</string>
-  <string name="attachproject_list_manual_dialog_title">Введите URL проекта:</string>
-  <string name="attachproject_list_manual_dialog_button">Добавить проект</string>
-  <string name="attachproject_list_manual_no_url">Пожалуйста, введите URL проекта</string>
-  <string name="attachproject_list_acctmgr_button">Добавить менеджер проектов</string>
+    <string name="attachproject_list_acctmgr_button">Добавить менеджер проектов</string>
```
Comment (adamradocz) [inline android/BOINC/app/src/main/res/values-ru/strings.xml:13]: Please fix the indentation.

**Comment by AenBleidd** [comment]
Not needed anymore

**Pull request closed without merging**